esp-hal/src/i2s/master/mod.rs

```diff
@@ -834,6 +837,29 @@ mod private {
         fn din_signal(&self) -> InputSignal;
     }
 
+    pub trait ExtendedSignals: Signals {
```

Comment:
CI currently complains since this generates warnings for any non-`esp32`-cfg builds

should this trait have a `#[cfg(esp32)]`? or is there a (reasonable) way to implement it for all chip types?

while the esp32s2 & esp32s3 also have e.g. the `I2S0I_WS` signal the same isn't true for e.g. `I2S0I_DATA_0`, thus i'm not sure if any other chip would support this (and just has different names for these signals)?

Reply:
Just cfg it for now